nerfstudio/scripts/datasets/process_project_aria.py

```diff
@@ -118,25 +119,103 @@ def read_trajectory_csv_to_dict(file_iterable_csv: str) -> TimedPoses:
     )
 
 
+def undistort_image_and_calibration(
+    input_image: np.ndarray,
+    input_calib: calibration.CameraCalibration,
+    output_focal_length: int,
+) -> [np.ndarray, calibration.CameraCalibration]:
```

Comment:
is this a valid annotation? should it be `Tuple[np.ndarray, calibration.CameraCalibration]`?